### PR TITLE
feat(event_list): change tooltip style

### PR DESCRIPTION
**Pull request by dqunbp**
change disaster card stats tooltip style
https://kontur.fibery.io/Tasks/Task/Change-tooltips-style-on-disaster-card-14451

### CODE REVIEW

**Comment by IlyaIzr** [comment]
1. Task says "Add tooltip «Open timeline» for play button" - i don't see that on the code
2. Branch doesn't run locally 
image

Code that is presented so far looks good

**Comment by propakov** [comment]
@IlyaIzr on my pc runs ok locally

File: src/features/events_list/components/EpisodeTimelineToggle/EpisodeTimelineToggle.tsx [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+        ref={ref}
+        className={s.timelineIcon}
+        onClick={panelState.isOpen ? panelStateActions.close : panelStateActions.open}
+        onMouseEnter={() => setShowTooltip(true)}
+        onMouseLeave={() => setShowTooltip(false)}
+      >
+        {isActive && panelState.isOpen ? <PlayActive24 /> : <Play24 />}
+      </span>
+      <Tooltip open={showTooltip} hoverBehavior>
+        <Trans i18nKey="event_list.open_timeline_button">Open timeline</Trans>
```
Comment (propakov) [inline src/features/events_list/components/EpisodeTimelineToggle/EpisodeTimelineToggle.tsx]: hm, `<Trans />` should only be used if there is html content inside translation string
here it seems that you need just {i18n.t("event_list.open_timeline_button")}

**Comment by propakov** [comment]
Have lots of warnings in console when just open the map without any actions
And timeline's 'Play button' tooltip not working
Screenshot 2023-02-17 at 09 25 22

**Comment by propakov** [comment]
could you also please make SEVERITY_TO_TEXT as a function that returns translation, or put this const inside the component? 
i18.n should be called from component, not just on app start
now it's a bug (try to show tooltip after lng switch): 
Screenshot 2023-02-17 at 09 42 55

**Comment by dqunbp** [comment]
> could you also please make SEVERITY_TO_TEXT as a function that returns translation, or put this const inside the component? i18.n should be called from component, not just on app start now it's a bug (try to show tooltip after lng switch): Screenshot 2023-02-17 at 09 42 55

Now we don't have a translation for this

**Comment by dqunbp** [comment]
> Have lots of warnings in console when just open the map without any actions And timeline's 'Play button' tooltip not working Screenshot 2023-02-17 at 09 25 22

Fixed, `targetRef` prop was skipped

**Comment by propakov** [comment]
> > could you also please make SEVERITY_TO_TEXT as a function that returns translation, or put this const inside the component? i18.n should be called from component, not just on app start now it's a bug (try to show tooltip after lng switch): Screenshot 2023-02-17 at 09 42 55
> 
> Now we don't have a translation for this

yeah, but it will work after translation appear, so no problem here 👍

File: src/components/SeverityIndicator/SeverityIndicator.tsx [hunk 2]
```diff
@@ -2,23 +2,24 @@ import { Tooltip } from '@konturio/ui-kit';
...
+    UNKNOWN: i18n.t('event_list.severity_unknown'),
+    TERMINATION: i18n.t('event_list.severity_termination'),
+    MINOR: i18n.t('event_list.severity_minor'),
+    MODERATE: i18n.t('event_list.severity_moderate'),
+    SEVERE: i18n.t('event_list.severity_severe'),
+    EXTREME: i18n.t('event_list.severity_extreme'),
+  };
+
+  return translations[severity];
 };
```
Comment (propakov) [inline src/components/SeverityIndicator/SeverityIndicator.tsx]: just not to call all translations even if they not needed: 
```suggestion
const severityToText = (severity: Severity) => {
  switch (severity) {
    case 'UNKNOWN':
      return i18n.t('event_list.severity_unknown');
    case 'TERMINATION':
      return i18n.t('event_list.severity_termination');
    case 'MINOR':
      return i18n.t('event_list.severity_minor');
    case 'MODERATE':
      return i18n.t('event_list.severity_moderate');
    case 'SEVERE':
      return i18n.t('event_list.severity_severe');
    case 'EXTREME':
      return i18n.t('event_list.severity_extreme');
  }
};
```

**Review by propakov — APPROVED** [review]
Works good 👍 
Found one minor improvement

File: src/components/SeverityIndicator/SeverityIndicator.tsx (hunk 2 above)
Comment (dqunbp) [reply src/components/SeverityIndicator/SeverityIndicator.tsx]: I thought about switch-case but decided that the object be a more compact construction

**Pull request merged**